libfuse/fs_darwin.go

```diff
@@ -43,3 +106,39 @@ func bundleResourcePath(path string) (string, error) {
 	}
 	return filepath.Join(execPath, "..", "..", "..", "Resources", path), nil
 }
+
+// Trash is a mock .Trashes directory
```

Comment:
Please explain that for now it just links to a directory under the user's private home directory, and a TODO for how we might fix this in the future.